src/pages/resultsView/cnSegments/CNSegments.tsx

```diff
@@ -52,10 +52,9 @@ export default class CNSegments extends React.Component<{ store: ResultsViewPage
     }
 
     @computed get chromosome() {
-        const gene = this.props.store.genes.result ?
-            this.props.store.genes.result.find(g => g.hugoGeneSymbol === this.activeLocus) : undefined;
-
-        return gene ? gene.chromosome : undefined;
+        if (this.props.store.hugoGeneSymbolToReferenceGene.result) {
```

Comment:
generally in this case we would want to use a remoteData - await hugoGeneSymbolToReferenceGene and then use it. unless it genuinely doesnt matter and `undefined` is an equally valid value. In the latter case, you would still need to `return undefined` explicitly